flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/nodes/exec/stream/WindowTableFunctionJsonPlanTest_jsonplan/testIndividualWindowTVFProcessingTime.out

```diff
@@ -173,6 +173,9 @@
   }, {
     "id" : 3,
     "type" : "stream-exec-watermark-assigner_1",
+    "configuration" : {
+      "table.exec.source.idle-timeout" : "0 ms"
```

Comment:
I don't think that this changes the topology. We can remove it from the consumed options.

Reply:
Also can we check one more time whether `table.exec.state.ttl` should be in the plan. I think we can leave this mutable.

Reply:
I think it's important, it affects `WatermarkAssignerOperator`, not the topology, but runtime how the watermarks are emitted, what do you think?

Reply:
And the `table.exec.state.ttl` is used for the `StateTtlConfig` which is "vital" for the relevant operations.

Reply:
`StateTtlConfig` is definitely a runtime configuration. If a user observes that the state size of the query grows over time, it should still be possible to change the configuration. The new configuration will be directly forwarded to the state backend and does not affect the SQL operator topology at all. Where does it affect WatermarkAssignerOperator?

Reply:
I mean the `table.exec.source.idle-timeout` affects the `WatermarkAssignerOperator`.